packages/toc/src/registry.ts

```diff
@@ -162,9 +162,10 @@ export namespace TableOfContentsRegistry {
      * -   If not present, a default renderer will be used.
      *
      * @param item - heading
+     * @param toc - list of headings
      * @returns JSX element
      */
-    itemRenderer?: (item: IHeading) => JSX.Element | null;
+    itemRenderer?: (item: IHeading, toc: IHeading[]) => JSX.Element | null;
```

Comment:
In this function, the `toc` argument is also a backward-incompatible signature change. Is it possible to make this one optional, as well?

Reply:
This should be resolved now as well.